variables.tf

```
@@ -174,6 +174,12 @@ variable "share_private_subnets" {
   description = "If set it will share the private subnets through resource access manager"
 }
 
+variable "shared_public_route_table" {
+  description = "Controls if public subnet routing will be handled by a shared route table or one table per subnet"
+  type        = bool
+  default     = true
+}
```

Slightly updated the description, but most importantly updated (fixed) the ordering:

```suggestion
variable "shared_public_route_table" {
  type        = bool
  default     = true
  description = "Determines weather to use a single route table for all public networks"
}
```

And following alphabetical ordering, this variable should be below `share_public_subnets`.